src/hypothesis/strategies.py

```diff
@@ -927,11 +927,18 @@ def from_type(thing):
     """
     from hypothesis.searchstrategy import types
     if not isinstance(thing, type):
-        # Under Python 3.6, Unions are not instances of `type` - but we still
-        # want to resolve them!  This __origin__ check only passes if thing is
-        # a Union with parameters; if it doesn't we can't resolve it anyway.
         try:
+            # At runtime, `typing.NewType` returns the identity function rather
```

Comment:
Nitpick: Maybe "an identity function" as it's not some canonical value but instead has these special attributes.